sql_driver/lib.go

```diff
@@ -16,7 +19,17 @@ func (d *LibsqlDriver) Open(dbPath string) (driver.Conn, error) {
 	if strings.HasPrefix(dbPath, "file:") {
 		return (&sqlite3.SQLiteDriver{}).Open(dbPath)
 	}
-	return sqlddriver.SqldConnect(dbPath), nil
+	if strings.HasPrefix(dbPath, "wss://") {
+		jwt := os.Getenv("SQLD_AUTH_TOKEN")
+		if len(jwt) == 0 {
+			return nil, fmt.Errorf("missing authorization token. Please provide JWT by setting SQLD_AUTH_TOKEN env variable")
+		}
```

Comment:
The JWT should be optional, for example if you want to connect to a local `sqld` instance.

Reply:
done.